### PR TITLE
ClimateEmergency_Toolkit: Compliance adjustments

### LINKED ISSUE #38: ClimateEmergency_Toolkit: Make compliant for installer inclusion
Some broken compliance rules that prevent this toolkit from being included in installers needs rectifying. List is as below.

 - [x] Copyright should be for BHoM not 2019 in AssemblyInfo.cs
 - [x] Input attribute should start with a lowercase letter
    in ClimateEmergency_Engine\Compute\AcidificationPotential.cs at 42:81
 - [x] Method must contain an Output or MultiOutput attribute
    in ClimateEmergency_Engine\Compute\AcidificationPotential.cs at 42:29
 - [x] Input attribute should start with a lowercase letter
    in ClimateEmergency_Engine\Compute\DepletionofAbioticResources.cs at 42:86
 - [x] Method must contain an Output or MultiOutput attribute
    in ClimateEmergency_Engine\Compute\DepletionofAbioticResources.cs at 42:29
 - [x] Input attribute should start with a lowercase letter
    in ClimateEmergency_Engine\Compute\DepletionofAbioticResourcesFossilFuels.cs at 42:97
 - [x] Method must contain an Output or MultiOutput attribute
    in ClimateEmergency_Engine\Compute\DepletionofAbioticResourcesFossilFuels.cs at 42:29
 - [x] Engine Method must contain a Description attribute
    in ClimateEmergency_Engine\Compute\EasterlingNortherlingToLatLong.cs at 50:34
 - [x] Engine Method must contain a Description attribute
    in ClimateEmergency_Engine\Compute\EasterlingNortherlingToLatLong.cs at 55:34
 - [x] Engine Method must contain a Description attribute
    in ClimateEmergency_Engine\Compute\EasterlingNortherlingToLatLong.cs at 60:34
 - [x] Engine Method must contain a Description attribute
    in ClimateEmergency_Engine\Compute\EasterlingNortherlingToLatLong.cs at 65:34
 - [x] Engine Method must contain a Description attribute
    in ClimateEmergency_Engine\Compute\EasterlingNortherlingToLatLong.cs at 70:34
 - [x] Engine Method must contain a Description attribute
    in ClimateEmergency_Engine\Compute\EasterlingNortherlingToLatLong.cs at 75:32
 - [x] Engine Method must contain a Description attribute
    in ClimateEmergency_Engine\Compute\EasterlingNortherlingToLatLong.cs at 122:39
 - [x] Incorrect Engine class name based on file path
    in ClimateEmergency_Engine\Compute\EasterlingNortherlingToLatLong.cs at 217:21
 - [x] Input parameter requires a matching Input attribute
    in ClimateEmergency_Engine\Compute\EasterlingNortherlingToLatLong.cs at 122:64
 - [x] Input parameter requires a matching Input attribute
    in ClimateEmergency_Engine\Compute\EasterlingNortherlingToLatLong.cs at 122:80
 - [x] Files cannot contain more than one class
    in ClimateEmergency_Engine\Compute\EasterlingNortherlingToLatLong.cs at 217:21
 - [x] Method must contain an Output or MultiOutput attribute
    in ClimateEmergency_Engine\Compute\EasterlingNortherlingToLatLong.cs at 50:34
 - [x] Method must contain an Output or MultiOutput attribute
    in ClimateEmergency_Engine\Compute\EasterlingNortherlingToLatLong.cs at 55:34
 - [x] Method must contain an Output or MultiOutput attribute
    in ClimateEmergency_Engine\Compute\EasterlingNortherlingToLatLong.cs at 60:34
 - [x] Method must contain an Output or MultiOutput attribute
    in ClimateEmergency_Engine\Compute\EasterlingNortherlingToLatLong.cs at 65:34
 - [x] Method must contain an Output or MultiOutput attribute
    in ClimateEmergency_Engine\Compute\EasterlingNortherlingToLatLong.cs at 70:34
 - [x] Method must contain an Output or MultiOutput attribute
    in ClimateEmergency_Engine\Compute\EasterlingNortherlingToLatLong.cs at 75:32
 - [x] Method must contain an Output or MultiOutput attribute
    in ClimateEmergency_Engine\Compute\EasterlingNortherlingToLatLong.cs at 122:39
 - [x] Method name must match file name
    in ClimateEmergency_Engine\Compute\EasterlingNortherlingToLatLong.cs at 122:39
 - [x] Invalid Engine class: Engine classes must be static
    in ClimateEmergency_Engine\Compute\EasterlingNortherlingToLatLong.cs at 217:8
 - [x] Invalid Engine class name
    in ClimateEmergency_Engine\Compute\EasterlingNortherlingToLatLong.cs at 217:21
 - [x] Input attribute should start with a lowercase letter
    in ClimateEmergency_Engine\Compute\EutrophicationPotential.cs at 42:82
 - [x] Method must contain an Output or MultiOutput attribute
    in ClimateEmergency_Engine\Compute\EutrophicationPotential.cs at 42:29
 - [x] Input attribute should start with a lowercase letter
    in ClimateEmergency_Engine\Compute\GlobalWarmingPotential.cs at 42:81
 - [x] Method must contain an Output or MultiOutput attribute
    in ClimateEmergency_Engine\Compute\GlobalWarmingPotential.cs at 42:29
 - [x] Method name must match file name
    in ClimateEmergency_Engine\Compute\MaterialEndOfLifeTreatment.cs at 41:29
 - [x] Input attribute should start with a lowercase letter
    in ClimateEmergency_Engine\Compute\OzoneDepletionPotential.cs at 42:82
 - [x] Method must contain an Output or MultiOutput attribute
    in ClimateEmergency_Engine\Compute\OzoneDepletionPotential.cs at 42:29
 - [x] Input attribute should start with a lowercase letter
    in ClimateEmergency_Engine\Compute\PhotochemicalOzoneCreationPotential.cs at 42:94
 - [x] Method must contain an Output or MultiOutput attribute
    in ClimateEmergency_Engine\Compute\PhotochemicalOzoneCreationPotential.cs at 42:29
 - [x] Method name must match file name
    in ClimateEmergency_Engine\Compute\ValueInputs\DepletionofAbioticResourcesFossilFuels.cs at 44:29

### PR DESCRIPTION
**Pull request by michaelhoehn**
### Issues addressed by this PR

Closes #38 

-Update input variable case
-Added output attribute to all methods
-Updated file names to match method names

### CODE REVIEW

File: ClimateEmergency_Engine/Compute/LCA/AcidificationPotential.cs [hunk 1]
```diff
@@ -53,19 +54,19 @@ public static double AcidificationPotential(BHoMObject obj, CustomObject EPDData
...
+            if (epdData.CustomData.ContainsKey("Density"))
             {
-                density = (double)EPDData.CustomData["Density"];
+                density = (double)epdData.CustomData["Density"];
             }
             else
             {
                 BH.Engine.Reflection.Compute.RecordError("The EPDDataset must have a valid value for density under a 'Density' key.");
                 return 0;
             }
```
Comment (michaelhoehn) [inline ClimateEmergency_Engine/Compute/LCA/AcidificationPotential.cs]: @FraserGreenroyd I can use ```try catch``` with proper convert cast for all BHoM obj querying methods if needed. What do you recommend?

Comment (FraserGreenroyd) [reply ClimateEmergency_Engine/Compute/LCA/AcidificationPotential.cs]: I'm happy with a casting in this manner, but if you'd rather use `System.Convert.ToDouble` then I wouldn't be against it as it is slightly more obvious then what it's doing (casting may not be obvious to the novice coming into BHoM) :smile:

**Comment by michaelhoehn** [comment]
All checks passed and changes implemented. @FraserGreenroyd review and merge requested.

**Comment by michaelhoehn** [comment]
All checks passed and changes implemented. @FraserGreenroyd review and merge requested.

**Pull request merged**